### DISCUSSION TITLE
New version?

### DISCUSSION THREAD

**Discussion by Neustradamus** [LMMS/lmms#6604] [General]
Dear @lmms team,

It is possible to have a new release build?

Thanks in advance.

Regards.

**Comment by Monospace-V** [comment]
@Neustradamus ,
Unfortunately there is unlikely to be a stable release for at least about a year or so.
You may download the LMMS 1.3 alpha build from the [download page](https://lmms.io/download#windows) under "Nightly Versions," or a more recent one from any of the recent PRs.

I don't see any mention of a 1.4 ever possibly existing, and only mentions of a 1.3 and potential 2.0, but there are far too many things we're trying to fix within 1.3 itself and 2.0 could require a huge amounts of rewriting. What I am trying to say is that the future is so unclear we haven't even thought of "What next after 1.3?"
Given how I haven't seen a clear form of that question yet, I would give it at least two years before it's _released_ as stable, though I have seen some discussion on whether or not it's more stable than 1.2. There are still many things we're trying to fix, and I think we want them fixed for 1.3 itself.
If you want to, you can fix a few bugs and contribute to the next potential version of LMMS.

I agree it's been far too long, perhaps seeing some new versions could increase activity and not make LMMS seem as dead. It's been about a year since even the last nightly release.
Possibly a 1.3.x series... we do want to give our users a new stable with features like ghost notes, FX mixer colour-coding, and several improvements, I think. I don't know how I personally feel about a stable release of 1.3 just yet, but I believe it'd help the LMMS userbase increase, given how long we've kept our releases stagnant. (It does, though, feel a bit wrong to put it out anytime soon. But I don't develop, so I cannot comment on how much we've done.)

**Comment by QuetzalcoutlDev** [comment]
> @Neustradamus , Unfortunately there is unlikely to be a stable release for at least about a year or so. You may download the LMMS 1.3 alpha build from the [download page](https://lmms.io/download#windows) under "Nightly Versions," or a more recent one from any of the recent PRs.
> 
> I don't see any mention of a 1.4 ever possibly existing, and only mentions of a 1.3 and potential 2.0, but there are far too many things we're trying to fix within 1.3 itself and 2.0 could require a huge amounts of rewriting. What I am trying to say is that the future is so unclear we haven't even thought of "What next after 1.3?" Given how I haven't seen a clear form of that question yet, I would give it at least two years before it's _released_ as stable, though I have seen some discussion on whether or not it's more stable than 1.2. There are still many things we're trying to fix, and I think we want them fixed for 1.3 itself. If you want to, you can fix a few bugs and contribute to the next potential version of LMMS.
> 
> I agree it's been far too long, perhaps seeing some new versions could increase activity and not make LMMS seem as dead. It's been about a year since even the last nightly release. Possibly a 1.3.x series... we do want to give our users a new stable with features like ghost notes, FX mixer colour-coding, and several improvements, I think. I don't know how I personally feel about a stable release of 1.3 just yet, but I believe it'd help the LMMS userbase increase, given how long we've kept our releases stagnant. (It does, though, feel a bit wrong to put it out anytime soon. But I don't develop, so I cannot comment on how much we've done.)

I think they should not get discouraged, maybe LMMS has taken a long time to release a stable version, but that should not worry them, since we as users will be waiting for the next version, the 1.3 alpha showed us that LMMS will be a good software , thanks for your effort @LMMS

By the way I have been using LMMS 1.3 alpha for months and the truth is it is quite stable and more than 1.2.2, not if you can take that as motivation coming from a small and novice developer

**Comment by Veratil** [comment]
> I think they should not get discouraged, maybe LMMS has taken a long time to release a stable version, but that should not worry them, since we as users will be waiting for the next version, the 1.3 alpha showed us that LMMS will be a good software , thanks for your effort @LMMS
> 
> By the way I have been using LMMS 1.3 alpha for months and the truth is it is quite stable and more than 1.2.2, not if you can take that as motivation coming from a small and novice developer

I wouldn't say we're discouraged at the moment, only upset that it's taking much longer than we originally estimated to get 1.3 in a state we're comfortable with releasing. We're unfortunately on volunteer time, so it's impossible to really give a good estimate for large sweeping changes that we're trying to do. There have been hundreds of commits between the 1.2.2 release and master branch that we're excited to get out to everyone, but we're also trying to fix a whole slew of things that we keep finding as we're making the codebase easier to work on (for us and future developers).

I have asked about tagging a new 1.3 alpha version to put on the download page, but there are a few updates that we're looking to get in (mainly some RemoteVST updates) before that happens.

The best I can say for the next release is Soon:tm:. :)

**Comment by Rossmaxx** [comment]
> I have asked about tagging a new 1.3 alpha version to put on the download page, but there are a few updates that we're looking to get in (mainly some RemoteVST updates) before that happens.

How about releasing an alpha before and another one after the RemoteVST updates? The entire community is waiting for it and i remember pushing it atleast a million times in the discord.

**Comment by Veratil** [comment]
@DomClark requested to push his fixes before that. Any ETA on those, Dom?

I'm with you in wanting a new alpha, but personally I'd rather wait and produce a single build because 1) Dom's fix is an important fix that I believe is worth waiting for, and 2) we still have to manually make builds at the moment and PhysSong is the one who makes those, and if we did a before/after that makes him go through the whole process twice (build for all platforms, gather changelog, update lmms.io; it's a bit of work still).

I know we're at a very slow pace right now, but even for an alpha we should want to treat it as a quasi-stable build since a lot of users will update to it. We're 1.5 years since previous alpha build, and 2.25 years since stable, delaying a build just a little longer will be fine (and worth it 😉).

**Comment by eagles051387** [comment]
Hi Kevin,

A question is there no way that the build process can be automated at this point that way at least nightly builds or once weekly builds can be rolled out for all platforms?

Regards,
Jonathan Aquilina

**Comment by Veratil** [comment]
> Hi Kevin, A question is there no way that the build process can be automated at this point that way at least nightly builds or once weekly builds can be rolled out for all platforms? Regards, Jonathan Aquilina

There is progress towards this being done. A few other devs have put in some good work on CI scripts to check that PRs don't break anything. We're getting closer to automated builds, but we're not quite there just yet.

**Comment by Rossmaxx** [comment]
Closing as duplicate of other similar discussions.

**Comment by Neustradamus** [comment]
@Rossmaxx: Where duplicates?

**Reply by Rossmaxx** [reply]
#6777

**Reply by Rossmaxx** [reply]
Even without the duplicate, the question is annoying to hear for us devs. I'm already a bit frustrated about the slow release (I used to be when i started volunteering but now i got used to it). There are chances we might get atleast an alpha in 2024.

**Comment by Neustradamus** [comment]
@Rossmaxx: You must to reopen this ticket and close as duplicate the other one...

**Reply by zonkmachine** [reply]
@Neustradamus Please, we're inching our way to a new release 1.3 
We, the developers, do it on our own timetable and the discussions happen in the developer forum.
This discussion here is just a distraction and therefore now closed.

In the mean time you can try out the 'Nightly' releases which are the latest builds and up to date.
https://lmms.io/download#linux
